addons/crm/tests/test_crm_lead_convert.py

```diff
@@ -250,7 +251,6 @@ def test_lead_merge_duplicates(self):
         self.lead_1.write({
             'partner_id': self.customer.id,
         })
-        self.customer.write({'email': False})
```

Comment:
Don't change tests. Actually ensure tests acts exactly like before; if we update some lead email we should not propagate it to customer (like resetting the custome email to previous value) because we need to keep I/O the same.

Reply:
@tde-banana-odoo 
But the task is about that, if I update the email/phone of a lead, and if a partner is set, it should write on the partner 
(that's the reason why I need to update the tests)

Reply:
Here you remove an unrelated change. We have to test existing data, notably if you ahve a customer with falsy email.

Reply:
> no, it's related, because I write on the partner, it will change also the email of the lead


It should not work that way.

Change lead's email -> update partner one (new spec, ok, not fan but why not)
Update partner's email -> do not touch leads (because discussion with customer may already have began and you should not change it)

Lead have priority on generic customer information.

Reply:
(I added a message on the task to include lna in the discussion ;) )